network-api/networkapi/wagtailpages/models.py

```diff
@@ -52,8 +52,9 @@
     ('pulse_listing', customblocks.PulseProjectList()),
     ('profile_listing', customblocks.LatestProfileList()),
     ('profile_by_id', customblocks.ProfileById()),
+    ('profile_directory', customblocks.ProfileDirectory()),
```

Comment:
ammending this:
```suggestion
   ('profile_directory', customblocks.ProfileDirectory()) if settings.ENABLE_PROFILE_DIRECTORY_BLOCK else None,
```
and then adding `ENABLE_PROFILE_DIRECTORY_BLOCK=True` to your .env file should show the profile directory in the CMS streamfield component picker, and setting it to `False` or omitting it from your .env file should show it _not_ being in the CMS streamfield component picker now.

Reply:
Got it. 👍